lib/apiservers/engine/backends/container_proxy.go

```diff
@@ -1470,13 +1470,11 @@ func processVolumeFields(volumes []string) (map[string]volumeFields, error) {
 }
 
 func finalizeVolumeList(specifiedVolumes, anonymousVolumes []string) ([]volumeFields, error) {
-	log.Infof("Specified Volumes : %#v", specifiedVolumes)
```

Comment:
I would rather see this logging changed to debug instead of removed as it indicates the volume requirements requested of the personality during a create call.